src/main/groovy/org/jenkinsci/gradle/plugins/jpi/JpiPlugin.groovy

```diff
@@ -177,6 +177,7 @@ class JpiPlugin implements Plugin<Project>, PluginDependencyProvider {
             t.upstreamManifest.set(jenkinsManifest.get().outputFile)
             t.description = 'Generate hpl (Hudson plugin link) for running locally'
             t.group = 'Jenkins Server'
+            t.dependsOn(gradleProject.tasks.getByName('processResources'))
```

Comment:
The dependency should be handled by declaring the relevant file input to the task, not by adding a dependsOn. Or is the input file not necessary?